test/extended/operators/scheduling.go

```diff
@@ -0,0 +1,48 @@
+package operators
+
+import (
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = Describe("[Feature:Platform][Smoke] Managed cluster should", func() {
+	f := e2e.NewDefaultFramework("operators")
+
+	It("should ensure control plane pods specify a priority", func() {
```

Comment:
The `Describe()` and the `It()` both have a "should". You're going to end up with "Managed cluster should should ensure..."